### PR TITLE
Coverage

### PR DESCRIPTION
**Pull request by Janther**
This PR solves an issue with try/catch discovered by adding more coverage cases and ups the coverage of `src/nodes` to a 100%

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -21,9 +21,7 @@ function parse(text, parsers, options) {
       if (ctx.initExpression) {
         ctx.initExpression.omitSemicolon = true;
       }
-      if (ctx.loopExpression) {
-        ctx.loopExpression.omitSemicolon = true;
-      }
+      ctx.loopExpression.omitSemicolon = true;
```
Comment (mattiaerre) [inline src/parser.js]: I guess that w/in `ForStatement`s we always have a `ctx.loopExpression` right?

File: src/nodes/CatchClause.js [hunk 2]
```diff
@@ -6,15 +6,22 @@ const {
 
 const printList = require('./print-list');
 
+const parameters = (node, path, print) =>
```
Comment (mattiaerre) [inline src/nodes/CatchClause.js]: just noticing this right now, do we always use function expressions?

File: src/parser.js (hunk 1 above)
Comment (Janther) [reply src/parser.js]: Yeah, the initial expression and the condition expression can be `null` but for some reason the loop expression is an expression with `null` content.
Maybe we can ask for some consistency but right now that’s how it is

File: src/nodes/CatchClause.js (hunk 2 above)
Comment (Janther) [reply src/nodes/CatchClause.js]: I tend to favour this notation. Is there an improvement by using declarations instead?

Comment (mattiaerre) [reply src/nodes/CatchClause.js]: not really, but to me, if it's a function I like to signal that w/ the `function` keyword

**Pull request merged**